src/components/ReportActionItem/ReportActionItemImage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import RenderHTML from '../RenderHTML';
+import Image from '../Image';
+
+const propTypes = {
+    /** thumbnail URI for the image */
+    thumbnail: PropTypes.string,
+
+    /** URI for the image */
+    image: PropTypes.string.isRequired,
+
+    /** whether or not to enable the image preview modal */
+    enablePreviewModal: PropTypes.bool,
+};
+
+const defaultProps = {
+    thumbnail: null,
+    enablePreviewModal: false,
+};
+
+function ReportActionItemImage({thumbnail, image, enablePreviewModal}) {
+    if (thumbnail) {
+        return (
+            <RenderHTML
+                html={`
+                    <img
+                        src="${thumbnail}"
+                        data-expensify-source="${image}"
+                        data-expensify-fit-container="true"
+                        data-expensify-preview-modal-disabled="${!enablePreviewModal}"
+                    />
+            `}
+            />
```

Comment:
I agree that ThumbnailImage would simplify ReportPreview and MoneyRequestPreview, but we would probably use RenderHTML for the MoneyRequestView.

Reply:
Is this due to `enablePreviewModal`? It may be best to move the logic in `ImageRenderer` to another component and use it there and in here

(not a blocker - possibly a follow up)

Reply:
I'd also agree, but probably a follow up where we can move the logic out and remove `data-expensify-fit-container`. For now, let's go with this.